application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -93,6 +102,9 @@ export class ViewSearchInput
                     this.active = new ActiveSearch(filter);
                     this.input.drop();
```

Comment:
better setup `this.active` with event `finish` (event context should be changed to reflect results: `{ found: number; error?: string; filter: IFilter}`